components/Starknet/modules/quick_start/pages/interact_with_a_smart_contract_using_remix.adoc

```diff
@@ -0,0 +1,285 @@
+= Interacting with a smart contract with Remix
+
+We shall utilize the Starknet Remix Plugin for the compilation, deployment, and interaction with our smart contract. This tool proves invaluable for initiating Starknet development, obviating the necessity of any local installations on your machine.
+
+== Introduction
+
+. Access the https://remix.ethereum.org/#activate=Starknet&lang=en&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.24+commit.e11b9ed9.js)[Remix IDE] website with the Starknet plugin activated.
+. Navigate to the `settings` option and opt for the Cairo version, as depicted in the accompanying image. The latest iteration available in Remix is `v2.5.4`.
```

Comment:
>  as depicted in the accompanying image.
What image?
>  The latest iteration available in Remix is `v2.5.4`.
Remove this. It's going to be obsolete as soon as Remix updates this.

Reply:
You gave me the cue for a doubt that came up, in the original repository screenshots were used, but they were out of context. I'm working on new images to fit the repository.